locale/pt_BR/locale.xml

```diff
@@ -130,7 +130,7 @@
 	<message key="submission.submissionEditing">Edição da submissão</message>
 	<message key="submission.changeSection">Alterar para</message>
 	<message key="submission.editorDecision">Decisão Editorial</message>
-	<message key="submission.round"><![CDATA[Rodada&nbsp;{$round}]]></message>
+	<message key="submission.round"><![CDATA[Rodada {$round}]]></message>
```

Comment:
&nbsp; looks noisy for non IT translators... removing the &nbsp; did not compromised final results...